### PR TITLE
Staticcheck: vendor/k8s.io/kubectl/pkg/cmd/exec|config|certificates

### PR DESCRIPTION
**Pull request by gavinfish**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Fix static check failures for:

```
vendor/k8s.io/kubectl/pkg/cmd/certificates
vendor/k8s.io/kubectl/pkg/cmd/config
vendor/k8s.io/kubectl/pkg/cmd/exe
```
**Which issue(s) this PR fixes**:

Ref: #81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
None
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
None
```

### CODE REVIEW

**Review by fejta — COMMENTED** [review]
/lgtm
/approve

File: staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go [hunk 1]
```diff
@@ -234,7 +234,7 @@ func (o *CertificateOptions) modifyCertificateCondition(builder *resource.Builde
 			csr := info.Object.(*certificatesv1beta1.CertificateSigningRequest)
 			csr, hasCondition := modify(csr)
```
Comment (fejta) [inline staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go]: This can also be `_, hasCondition`

Comment (gavinfish) [reply staging/src/k8s.io/kubectl/pkg/cmd/certificates/certificates.go]: Seems like that, but I prefer to keep it like that for the design of the modify function. Some one may construct a totally new `certificatesv1beta1.CertificateSigningRequest` object :thinking:

**Review by soltysh — APPROVED** [review]
/lgtm
/approve

**Pull request merged**